src/screens/Classroom/CreateClassroomScreen.js

```diff
@@ -0,0 +1,152 @@
+import 'react-native-gesture-handler';
+import React, { useState, useEffect } from 'react';
+// import { NavigationContainer } from '@react-navigation/native';
+// import { createStackNavigator } from '@react-navigation/stack';
+import {
+  Text, View, StyleSheet, Alert, Button, TextInput,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import Firestore, { firebase } from '@react-native-firebase/firestore';
+// import Post from './Post';
+import { INITIAL_USER_STATE } from '../../components';
+
+const styles = StyleSheet.create({
+  container: {
+    marginTop: 10,
+    marginBottom: 10,
+    marginHorizontal: 10,
+    padding: 10,
+    backgroundColor: '#ffffff',
+    borderRadius: 10,
+  },
+  contentContainer: {
+    paddingTop: 20,
+  },
+  timeText: {
+    fontSize: 11,
+  },
+  topicText: {
+    fontSize: 18,
+    fontWeight: 'bold',
+  },
+  bodyText: {
+    fontSize: 14,
+  },
+});
+
+export default function CreateClassroomScreen({ navigation }) {
+  const [title, setTitle] = useState('');
+  const [initializing, setInitializing] = useState(true);
+  const uid = navigation.getParam('uid', null);
+  const ref = Firestore().collection('users');
+  const [userState, setUserState] = useState(INITIAL_USER_STATE);
+  const [errorMessage, setErrorMessage] = useState(null);
+
+  async function getUserData() {
```

Comment:
Our code base uses the promise chaining function style - can you convert this code the same way I did in ClassroomHome.js (although ClassroomHome.js may not be used for now, simply disconnect it from other code rather than deleting it complete for now).